spec/unit/shoulda/matchers/action_controller/callback_matcher_spec.rb

```diff
@@ -47,10 +47,90 @@ def match
       end
     end
 
+    context "qualified with only" do
+      let(:action) { :index }
+      let(:multiple_actions) { [:index, :show] }
+      let(:other_action) { :other_action }
+
+      context "and when a #{kind} hook is in place with the qualifier" do
+        it "accepts" do
+          add_callback(kind, callback_type, method_name, only: action)
+          expect(controller).to match.only(action)
+        end
+
+        it "accepts with multiple qualifier options" do
+          add_callback(kind, callback_type, method_name, only: multiple_actions)
+          expect(controller).to match.only(multiple_actions)
+        end
+      end
+
+      context "and when a #{kind} hook is in place without the qualifier" do
+        it "rejects" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.